conf/battle/client.conf

```diff
@@ -173,3 +173,7 @@ macro_detection_punishment_time: 0
 // Macrochecker delay (per map)
 // Set to 0 to disable
 macrochecker_delay: 600000
+
+// Should companion pets/elementals/homunculi/mercenaries display the guild emblem of their master? (Note 3)
+// Default: 0 (none)
+companion_show_guild_emblem: 0
```

Comment:
Here is actually another thought... why don't we call this option just "show_guild_emblem" and set the default to 1. Then users could also configure if the emblem should be shown on the player. Or is it not possible to not show it on PC?

Reply:
> Here is actually another thought... why don't we call this option just "show_guild_emblem" and set the default to 1. Then users could also configure if the emblem should be shown on the player. Or is it not possible to not show it on PC?

in current status.cpp there isnt bl)->master at case BL_PC